packages/blade/src/components/Chip/chipTokens.ts

```diff
@@ -0,0 +1,209 @@
+import type { ChipGroupProps } from './ChipGroup';
+import type { Theme } from '~components/BladeProvider';
+import type { DotNotationColorStringToken, DotNotationSpacingStringToken } from '~utils/types';
+import type { SelectorInputHoverTokens } from '~components/Form/Selector/types';
+import { size } from '~tokens/global';
+import type { IconProps } from '~components/Icons';
+import type { BaseTextProps } from '~components/Typography/BaseText/types';
+
+const chipGroupGapTokens = {
+  xsmall: {
+    right: 'spacing.3',
+    bottom: 'spacing.3',
+  },
+  small: {
+    right: 'spacing.3',
+    bottom: 'spacing.3',
+  },
+  medium: {
+    right: 'spacing.3',
+    bottom: 'spacing.4',
+  },
+  large: {
+    right: 'spacing.3',
+    bottom: 'spacing.4',
+  },
+} as const;
+
+const chipHeightTokens: Record<NonNullable<ChipGroupProps['size']>, number> = {
+  xsmall: size[24],
+  small: size[28],
+  medium: size[36],
+  large: size[48],
+};
+
+type ChipHorizontalPaddingTokens = {
+  default: Record<
```

Comment:
maybe rename it to `withoutIcon` and `withIcon`? right now its not very clearly understandable what does default mean in this context

Reply:
Updated.